catalog/resource_registered_model.go

```diff
@@ -0,0 +1,78 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceRegisteredModel() *schema.Resource {
+	s := common.StructToSchema(
+		catalog.CreateRegisteredModelRequest{},
+		func(m map[string]*schema.Schema) map[string]*schema.Schema {
+			m["name"].ForceNew = true
+			m["catalog_name"].ForceNew = true
+			m["schema_name"].ForceNew = true
+			m["storage_location"].ForceNew = true
```

Comment:
If it's computed, we may not need `ForceNew`.

Reply:
Yeah, but if someone manually sets this field after it's been computed, it still tries to modify the field in-place instead of replacing the whole resource. That's why I added both attributes